app/__tests__/AddRuleModal.test.jsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { shallow } from "enzyme";
+import { AddRuleModal } from "./../components/AddRuleModal";
+
+const createTestProps = props => ({
+  handleClose: jest.fn(),
+  showModal: false,
+  modalMethod: "GET",
+  addRequest: jest.fn(),
+  modalUrl: "www.codemancers.com",
```

Comment:
Don't modal data based on UI. what if tomorrow we decided to not go with `Modal`? 
the name become meaningless and since data is used everywhere, renaming will impact a lot of places.

Reply:
Yes. ^^ that makes sense. Will change names.